Makefile

```diff
@@ -3,20 +3,31 @@ export GOPROXY=https://proxy.golang.org
 
 .PHONY: \
 	all \
+	binary \
 	clean \
+	codespell \
+	containers-storage \
+	cross \
 	default \
 	docs \
+	gccgo \
 	help \
+	install \
+	install.docs \
 	install.tools \
+	lint \
 	local-binary \
 	local-cross \
 	local-gccgo \
 	local-test \
 	local-test-integration \
 	local-test-unit \
 	local-validate \
-	lint \
-	vendor
+	test-integration \
+	test-unit \
+	validate \
+	vendor \
+	vendor-in-container
```

Comment:
I'm not sure we need to include vendor-in-container.

Reply:
Ya we do, it's a phony target - Meaning, it doesn't create a file called `vendor-in-container`.